marimo/_output/formatters/npndarray_formatters.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Marimo. All rights reserved.
+from __future__ import annotations
+
+from marimo._messaging.mimetypes import KnownMimeType
+from marimo._output.formatters.formatter_factory import FormatterFactory
+
+
+class NpndarrayFormatter(FormatterFactory):
+    @staticmethod
+    def package_name() -> str:
+        return "np.ndarray class type"
```

Comment:
this should be `numpy`

Reply:
Will rename both class name and return type to that.